services/github/github-r-package.tester.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const { isVPlusDottedVersionAtLeastOne } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+const mockContent = (path, versionLine) => nock => {
+  const content = [
+    'Package: mypackage',
+    'Title: What The Package Does (one line, title case required)',
+    versionLine,
+    '',
+  ].join('\n')
+
+  return nock('https://api.github.com')
+    .get(`/repos/mixOmicsTeam/mixOmics/contents/${path}`)
+    .query({ ref: 'HEAD' })
+    .reply(200, {
+      content: Buffer.from(content, 'utf-8').toString('base64'),
+      encoding: 'base64',
+    })
+}
```

Comment:
Similarly to my comment in https://github.com/badges/shields/pull/5705/files#r505937638, I think using standard unit tests against the `transform` function would probably be more lightweight and straightforward

Reply:
In the referenced comment I've shared my thought on that. Let's establish there the final way to go about those tests.

Reply:
Done.